style.css

```diff
@@ -0,0 +1,83 @@
+*{
```

Comment:
@Bishakha1589 Please redo this css using display, vw and vh in width and height and rem or em for font sizes. Avoid using pixel dimensions wherever possible.